GliaWidgets/Coordinator/Call/CallCoordinator.swift

```diff
@@ -63,7 +73,6 @@ class CallCoordinator: SubFlowCoordinator, FlowCoordinator {
                 self?.delegate?(.minimize)
             }
         }
-        return CallViewController(viewModel: viewModel,
-                                  viewFactory: viewFactory)
+        return CallViewController(viewModel: viewModel, viewFactory: viewFactory)
```

Comment:
39-42 lines have another alignment. Let's use the same formatting in both places.

Reply:
Lines 39-42 is how it should be, yes. I just thought that this line wasn't long enough to insert linebreaks. Corrected